config/routes.rb

```diff
@@ -24,6 +24,7 @@
 #                                               api_v2 GET    /api/v2/recruitment_cycles/:recruitment_cycle_year/providers/suggest(.:format)                                                           api/v2/providers#suggest
 #                                 api_v2_organisations GET    /api/v2/organisations(.:format)                                                                                                          api/v2/organisations#index
 #                   api_v2_provider_recruitment_cycles GET    /api/v2/providers/:provider_code/recruitment_cycles(.:format)                                                                            api/v2/recruitment_cycles#index
+#                          api_v2_provider_allocations POST   /api/v2/providers/:provider_code/allocations(.:format)                                                                                   api/v2/allocations#create
```

Comment:
As an example referring to my earlier appoint above ... (NB this is more of a general comment for discussion)
Looking at the new route that has been generated  its not immediately obvious was the relationship is between the allocation and the  Accrediting Body or Training Provider  - this knowledge is implicit, would be easier to get one's head around if it were more explicit in the code somehow?

Reply:
Yep, agreed as above.